.buildkite/pipeline.yml

```diff
@@ -18,5 +17,9 @@ steps:
     timeout: 100
 
   - label: "Run tests (bindists)"
-    command: ".buildkite/bindists-pipeline"
+    command: |
+      mkdir -p "$HOME/bindists/repo_cache"
+      mkdir -p "$HOME/bindists/.cache/bazel"
+      mkdir -p "$HOME/bindists/.stack"
+      docker run -it --network host -v $(pwd):/home/buildkite-agent/rules_haskell:rw -v $HOME/bindists/repo_cache:/home/buildkite-agent/repo_cache:rw -v $HOME/bindists/.cache/bazel:/home/buildkite-agent/.cache/bazel:rw -v $HOME/bindists/.stack:/home/buildkite-agent/.stack:rw --workdir /home/buildkite-agent/rules_haskell --user 1000 mrkkrp/rules_haskell:0.1.0 .buildkite/bindists-pipeline
```

Comment:
`mrkkrp/rules_haskell:0.1.0` are we planning to host the image under Tweag org at some point?

Reply:
This will soon be removed altogether. The image will be prepared here locally as a step in the pipeline.